fixtures/ibiza-light/pluton10-lzr.json

```diff
@@ -0,0 +1,497 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "pluton10-LZR",
+  "categories": ["Moving Head"],
+  "meta": {
+    "authors": ["Juditech3D"],
+    "createDate": "2024-02-02",
+    "lastModifyDate": "2024-02-02",
+    "importPlugin": {
+      "plugin": "qlcplus_4.12.1",
+      "date": "2024-02-02",
+      "comment": "created by Q Light Controller Plus (version 4.12.7)"
+    }
+  },
+  "physical": {
+    "dimensions": [159, 265, 147],
+    "weight": 3.4,
+    "power": 45,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED",
+      "colorTemperature": 6000,
+      "lumens": 5120
+    },
+    "lens": {
+      "name": "Fresnel"
+    }
+  },
+  "availableChannels": {
+    "Pan": {
+      "fineChannelAliases": ["Pan fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "Pan",
+        "angleStart": "0deg",
+        "angleEnd": "540deg"
+      }
+    },
+    "Tilt": {
+      "fineChannelAliases": ["Tilt fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "Tilt",
+        "angleStart": "0deg",
+        "angleEnd": "230deg"
+      }
+    },
+    "Color": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 8],
+          "type": "ColorPreset",
+          "comment": "Color 1"
+        },
+        {
+          "dmxRange": [9, 17],
+          "type": "ColorPreset",
+          "comment": "Color 1 +Color 2"
```

Comment:
Instead of ColorPreset, I think this should be Color Wheel (8 slots), and then reference it from a WheelSlot capability.  Then we can do half-n-half colors, too.